frontend/www/js/omegaup/problem/edit.ts

```diff
@@ -30,13 +30,30 @@ OmegaUp.on('ready', () => {
           initialAdmins: this.initialAdmins,
           initialGroups: this.initialGroups,
           initialLanguage: payload.statement.language,
-          log: this.log,
+          log: payload.log,
           publishedRevision: this.publishedRevision,
           value: this.publishedRevision,
           markdownContents: this.markdownContents,
           markdownSolutionContents: this.markdownSolutionContents,
+          problemLevel: this.problemLevel,
         },
         on: {
+          'update-problem-level': (levelTag?: string) => {
+            const params = levelTag
+              ? {
+                  problem_alias: payload.alias,
+                  level_tag: levelTag,
+                }
+              : {
+                  problem_alias: payload.alias,
+                };
+            api.Problem.updateProblemLevel(params)
```

Comment:
```suggestion
            api.Problem.updateProblemLevel({
                problem_alias: payload.alias,
                level_tag: levelTag,
            })
```

ah algo que se me olvidó preguntar: esto funciona? ahora en el API si se intenta mandar algo `null` o `undefined`, se omite del request, entonces esto _debería_ ser funcionalmente equivalente.

Reply:
Si funciona correctamente.